modules/theme-tailwind/molecules/AddToWishlist.vue

```diff
@@ -95,7 +95,10 @@ export default defineComponent({
         if (this.wishlisted) {
           await this.removeFromWishlist(this.$Product)
         } else {
-          await this.addToWishlist(this.$Product)
+          const { userErrors } = await this.addToWishlist(this.$Product)
```

Comment:
Dont change this code. Molecule should not care if the error is regular error or some special user error.